easybuild/easyconfigs/s/SAMtools/SAMtools-1.3.1_Makefile.patch

```diff
@@ -0,0 +1,24 @@
+Common subdirectories: samtools-1.3.1/examples and samtools-1.3.1.new/examples
+Common subdirectories: samtools-1.3.1/htslib-1.3.1 and samtools-1.3.1.new/htslib-1.3.1
```

Comment:
@verdurin please replace these lines with a short comment on what this patch does, and add yourself as an author